gprimitives/src/lib.rs

```diff
@@ -24,6 +24,7 @@
 #![doc(html_favicon_url = "https://gear-tech.io/favicons/favicon.ico")]
 
 pub use gear_ss58::Ss58Address;
+pub use primitive_types;
```

Comment:
what if explicitly reexport only dedicated and most-used for us types: H160, H256, U256.
Cause gprimitives::primitive_types looks anyway weird

Reply:
Should it be in root or in module?

Reply:
up to you
I'd go with root (like it's primitives for us as well)